src/Workleap.DomainEventPropagation.Subscription/Extensions/ServiceCollectionEventPropagationExtensions.cs

```diff
@@ -10,32 +10,16 @@ namespace Workleap.DomainEventPropagation.Extensions;
 public static class ServiceCollectionEventPropagationExtensions
 {
     public static IEventPropagationSubscriberBuilder AddEventPropagationSubscriber(this IServiceCollection services)
-        => services.AddEventPropagationSubscriber(_ => { });
-
-    public static IEventPropagationSubscriberBuilder AddEventPropagationSubscriber(this IServiceCollection services, Action<EventPropagationSubscriberOptions> configure)
```

Comment:
It was removed since the options class doesn't exists anymore and the options class doesn't exists anymore cause all it contained was a list of topic names to validate against the registered topic providers. But we don't have topic providers anymore so no need for those options either 😄